frontend/src/app/core/setup/globals/global-listeners/top-menu.ts

```diff
@@ -223,8 +223,8 @@ export class TopMenu {
 
   // If there is ANY input, it will have precedence over links,
   // i.e. links will only get focused, if there is NO input whatsoever
-  focusFirstInputOrLink(dropdown:JQuery) {
-    const focusable = dropdown.find('input:not([disabled]), a[href], area[href], select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, *[tabindex], *[contenteditable]');
+  focusFirstInputOrLink(dropdown:JQuery):void {
+    const focusable = dropdown.find('.op-app-menu--dropdown').find('input:not([disabled]):not([type="hidden"]), a[href], area[href], select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, *[tabindex], *[contenteditable]');
```

Comment:
Are you sure it's correct this will only find inside the dropdowns?

Reply:
The structure in the top menu is like this:
```
<li class="op-app-menu--item op-app-menu--item_has-dropdown last-child op-app-menu--item_dropdown-open"
  <a class="op-app-menu--item-action  " span_class="op-app-menu--item-title " href="#">
  </a>
  <ul id="user-menu" class="op-app-menu--dropdown op-menu " ></ul>
</li>
```
Without the extra condition it will always return this first link which is the one shown in the top menu (which triggered the drop down in the first place). So the way it was before, it actually always tried to focus the link I just clicked on and never any of the content in the dropdown.